src/commands/data/search.ts

```diff
@@ -33,10 +34,13 @@ export class DataSearchCommand extends SfCommand<SearchResult> {
       summary: messages.getMessage('flags.file.summary'),
       exactlyOne: ['query', 'file'],
     }),
-    'result-format': resultFormatFlag({
+    'result-format': Flags.custom<FormatTypes>({
+      char: 'r',
       summary: messages.getMessage('flags.result-format.summary'),
+      options: formatTypes,
+      default: 'human',
       exclusive: ['json'],
-    }),
+    })(),
```

Comment:
this was done to remove the deprecated `resultformat` alias, see the command snapshot diff